src/Documenter.jl

```diff
@@ -764,6 +769,8 @@ function git_push(
             isfile(keyfile) && rm(keyfile)
         end
     else # authentication_method(deploy_config) === HTTPS
+        # Remove SSH keys if existing
+        documenter_key_delete!()
```

Comment:
I don't think it's necessarily an issue, but does this break the case where you'd call `deploydocs` multiple times?

Reply:
Yea I guess, but will that matter in practice?

Reply:
I think we could live with it, but OTOH someone might create a weird setup where they deploy to multiple places or something like that. How doable do you think it would be to wrap everything in a big `withenv` instead?

Reply:
Yea maybe that is the easiest option.